statistics_collector_lib/include/collector/collector.hpp

```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
```

Comment:
the directory here should not be collector, it should be `libstatistics_collector`

Reply:
The directory structure has changed enough for this comment to no longer be applicable.